pages/_app.js

```diff
@@ -1,4 +1,5 @@
 import { GlobalStyle } from '../components/GlobalStyle';
+import './app.css';
 
```

Comment:
You shouldn't use external CSS files and `styled-components`. If you wish to use imported fonts, you might find this link useful for more information: https://nextjs.org/docs/basic-features/font-optimization :)

Reply:
Hey, thanks for the info. I've stumbled upon the next js doc yesterday but my _document.js already looks different because of the bable plugin (for ssr with styled components) and I don't know if I can just extend and/or change it. (see _document.js in files changed please)

Reply:
Removed the CSS, modified the _document and imported the font.
Thanks for the info.